jobs/elasticsearch_alerts/spec

```diff
@@ -0,0 +1,17 @@
+---
+name: elasticsearch_alerts
+
+packages: []
+
+templates:
+  cluster.alerts: cluster.alerts
+
+properties:
+  elastic_alerts.cluster_down.evaluation_time:
```

Comment:
This should be `elasticsearch_alerts.cluster_down.evaluation_time` to match what you have at the alert file.